package.json

```diff
@@ -25,6 +25,9 @@
     "url": "git+https://github.com/AndreMiras/edilkamin.js.git"
   },
   "author": "Andre Miras",
+  "contributors": [
+    { "name": "Maxime Journaux", "email": "maxime.journaux@espridigital.fr" }
```

Comment:
Not needed, you will automatically appear in the https://github.com/AndreMiras/edilkamin.js/graphs/contributors graph.
I'd rather keep it minimal, I don't think I have ever seen other packages deal with it this way